plugins/CoreHome/vue/src/Comparisons/Comparisons.store.ts

```diff
@@ -77,7 +77,9 @@ export default class ComparisonsStore {
   readonly isEnabled = computed(() => this.checkEnabledForCurrentPage());
 
   constructor() {
-    this.loadComparisonsDisabledFor();
+    document.addEventListener('DOMContentLoaded', () => {
+      this.loadComparisonsDisabledFor();
+    });
```

Comment:
This change was needed as `window.broadcast.isLoginPage()` checks if the body element has a certain id. But the javascript might currently already be parsed before the browser starts to render the html, which makes it impossible to look for html elements id.
Not sure if there is a smarter approach for this...

Reply:
just a thought that came up, but an alternative might be to make `Piwik::getLoginPluginName()` a global in jsGlobalVariables.twig and check if the URL module == that or `'Login'`